### ISSUE TITLE
Parse error when html tag is contained in body?

### ISSUE THREAD

**Issue by toshi0383** [shuntaka9576/gh-p2#3]
I encountered an error like this.

```
failed to run gh: gh: Parse error on "359" (INT) at [2, 331]
. error: exit status 1
```

I avoided this by temporary removing html tag from body, but should be able to parse correctly.

**Comment by shuntaka9576** [comment]
@toshi0383 
Thank you for the feedback!
I attempted to replicate the issue on my end, but I was unable to do so. If you could provide the details of the command that resulted in a parse error, I might be able to help fix it.


```bash
gh p2 create \
  --user "shuntaka9576" \
  -r "kanban-test" \
  -p "test" \
  --title "Fix bug" \
  --assignees "shuntaka9576" \
  --fields "Status:Todo" \
  --fields "point:3" \
  --fields "deadline:2022-08-11" \
  --body 'sample\n<details>\n<summary>\nSample Code</summary>\n<pre>\n<code>\necho Hello World\n</code>\n</pre></details>'
```

![image](https://github.com/shuntaka9576/gh-p2/assets/12817245/fc697af7-eae2-4768-9e4b-bb65d9718032)
![image](https://github.com/shuntaka9576/gh-p2/assets/12817245/072d4b3a-0d23-4670-957f-d0deb7289f3d)

**Comment by toshi0383** [comment]
@shuntaka9576 
I tested several times and here is the example.
Let me know if you need further information.
```
#!/bin/bash

# OK
BODY='テスト<div></div>'

# NG: failed to run gh: gh: Parse error on " />" (STRING) at [2, 114]
BODY='テスト<img src="" />'

gh p2 create \
    -o ... \
    -p ... \
    -r ... \
    --title ... \
    -f ...
    --body "$BODY"
```

**Comment by shuntaka9576** [comment]
@toshi0383 
Thank you for the feedback and sample code. I was able to reproduce the issue locally. 
The problem seemed to be due to a conflict between the double quotes in the GraphQL syntax. 
By escaping the double quotes, I was able to resolve the issue. #4

Please let me know if there's still any problems with the behavior 👍 

The issue has been addressed in version 0.1.2. Please update to this version to apply the fix.
```bash
gh extension upgrade shuntaka9576/gh-p2
$ gh p2 -v
p2 version 0.1.2 (rev:d2d7a97)
```
